hono/jest.config.js

```diff
@@ -1,5 +1,8 @@
-/** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
   preset: 'ts-jest',
-  testEnvironment: 'node',
+  testEnvironment: 'miniflare',
+  testEnvironmentOptions: {
+    bindings: { KEY: 'value' },
```

Comment:
What does this binding do?

Reply:
The `testEnvironmentOptions` is used for API options but since we do not have any I'll just remove them.